pkg/install/test-snapshot-class.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  name: test-snapshot-class
+driver: hostpath.csi.k8s.io
```

Comment:
I've manually created this `VolumeSnapshotClass` for my testing. I know that I should be getting the default CSI driver and creating this CR automatically but I'm not sure where in the runtime lifecycle that should happen.

Reply:
Doing this now when storage classes change and when the user runs `snapshot create`